deckhouse-controller/crds/module.yaml

```diff
@@ -131,10 +131,10 @@ spec:
       subresources:
         status: {}
       additionalPrinterColumns:
-        - name: Weight
```

Comment:
lets add `priority: 1` to the `weight` field. Sometimes it's useful to use --sort-by with weight field to watch the reconcile order of modules.